api/src/main/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceImpl.java

```diff
@@ -314,6 +314,35 @@ public String getReadOnlyNotebook(
         .collect(Collectors.toList());
   }
 
+  @Override
+  public void setAdminLockedState(String workspaceNamespace, boolean desiredLockState) {
+    // OK I got your request
+    log.info(
+        String.format(
+            "called setLockedState on wsns %s with desiredLockState %b",
+            workspaceNamespace, desiredLockState));
+
+    // TODO but what should I do with it?
+
+    Optional<DbWorkspace> dbWorkspaceOptional = getFirstWorkspaceByNamespace(workspaceNamespace);
```

Comment:
this version was intended to be lenient for prototyping.  For the actual implementation I prefer the `getFirstWorkspaceByNamespaceOrThrow()` call.